### PR TITLE
Update LLVM commit to 980d28d Feb 7

### PR DESCRIPTION
**Pull request by tungld**
Signed-off-by: Tung D. Le <tung@jp.ibm.com>

### CODE REVIEW

File: src/Dialect/ONNX/ONNXOps.cpp [hunk 1]
```diff
@@ -2068,7 +2068,7 @@ LogicalResult ONNXPadOp::inferShapes(
   SmallVector<int64_t, 4> outputShape(dataShape.begin(), dataShape.end());
 
   // Get pads from valueAttribute.
-  Attribute padattr = getAttr("pads");
+  Attribute padattr = (*this)->getAttr("pads");
```
Comment (tjingrant) [inline src/Dialect/ONNX/ONNXOps.cpp:2071]: Hi @tungld do you mind explaining why these patterns are popping up? i.e., why is the object `this` is referring to becomes something like a pointer to a pointer? Any patch/background discussion in mlir linked to this change?

Comment (tungld) [reply src/Dialect/ONNX/ONNXOps.cpp:2071]: Hi @tjingrant, sorry for my late reply!

I guess, `this` refers to `OpState::operator`. I found this MLIR patch which may help understand the change: https://github.com/llvm/llvm-project/commit/1ffc1aaa09674824ce66600ee0b77d6a2c042e68#diff-9185f7f3f3628467e62e1712b01efc482eecc3643b67746c1eae73f5fd272138

Comment (tjingrant) [reply src/Dialect/ONNX/ONNXOps.cpp:2071]: Got it, thanks! I have no concerns now.

**Review by tjingrant — APPROVED** [review]
LGTM!

**Pull request merged**